apps/web/.env.sample

```diff
@@ -50,6 +50,12 @@ APP_NAME='Ever Teams'
 APP_SIGNATURE='Ever Teams'
 APP_LOGO_URL='https://app.ever.team/assets/ever-teams.png'
 
+#Footer links
+NEXT_PUBLIC_EVER_TEAMS_LINK="https://gauzy.team"
+NEXT_PUBLIC_EVER_PLATFORME_LINK="https://ever.co"
```

Comment:
@samuelmbabhazi, replace NEXT_PUBLIC_EVER_PLATEFORME_LINK by NEXT_PUBLIC_EVER_PLATEFORM_LINK and Cspell error will be fixed